docs/deploy-full-rp-service-in-dev.md

```diff
@@ -0,0 +1,131 @@
+        --vault-name "$KEYVAULT_PREFIX-svc" \
+        --name rp-mdsd \
+        --file secrets/rp-logging-int.pem >/dev/null
+   az keyvault certificate import \
+        --vault-name "$KEYVAULT_PREFIX-svc" \
+        --name cluster-mdsd \
+        --file secrets/cluster-logging-int.pem >/dev/null
+    az keyvault certificate import \
+        --vault-name "$KEYVAULT_PREFIX-svc" \
+        --name dev-arm \
+        --file secrets/arm.pem >/dev/null
+   az keyvault certificate import \
+        --vault-name "$KEYVAULT_PREFIX-svc" \
+        --name rp-firstparty \
+        --file secrets/firstparty.pem >/dev/null
+    az keyvault certificate import \
+        --vault-name "$KEYVAULT_PREFIX-svc" \
+        --name rp-server \
+        --file secrets/localhost.pem >/dev/null
+    az keyvault certificate import \
+        --vault-name "$KEYVAULT_PREFIX-por" \
+        --name portal-server \
+        --file secrets/localhost.pem >/dev/null
+    az keyvault certificate import \
+        --vault-name "$KEYVAULT_PREFIX-por" \
+        --name portal-client \
+        --file secrets/portal-client.pem >/dev/null
+    ```
+
+
+1. Mirror the OpenShift images to your new ACR
+    <!-- TODO (bv) allow mirroring through a pipeline would be faster and a nice to have -->
+
+    1. Setup mirroring environment variables
+        ```bash
+        export DST_ACR_NAME=${USER}aro
+        export SRC_AUTH_QUAY=FILL_IN # Get quay auth https://cloud.redhat.com/openshift/create/local -> Download Pull Secret
+        export SRC_AUTH_REDHAT=$(echo $USER_PULL_SECRET | jq -r '.auths."registry.redhat.io".auth')
+        export DST_AUTH=$(echo -n '00000000-0000-0000-0000-000000000000:'$(az acr login -n ${DST_ACR_NAME} --expose-token | jq -r .accessToken) | base64 -w0)
+
+    1. Login to the Azure Container Registry
+        ```bash
+        docker login -u 00000000-0000-0000-0000-000000000000 -p "$(echo $DST_AUTH | base64 -d | cut -d':' -f2)" "${DST_ACR_NAME}.azurecr.io"
+        ```
+
+    1. Run the mirroring
+        > The `latest` argument will take the InstallStream from `pkg/util/version/const.go` and mirror that version
+        ```bash
+        go run ./cmd/aro mirror latest
+        ```
+
+    1. Push the ARO image to your ACR
+        ```bash
+        make publish-image-aro-multistage
+        ```
+
+1. Delete the existing VMSS
+    > __NOTE:__ This needs to be deleted as deploying won't recreate the VMSS if the commit hash is the same.
+    ```bash
+    az vmss delete -g ${RESOURCEGROUP} --name rp-vmss-$(git rev-parse --short=7 HEAD)$([[ $(git status --porcelain) = "" ]] || echo -dirty)
+    ```
+
+1. Run `make deploy`
+
+## Deploying a cluster
+
+1. Setup a local tunnel to the RP
+    ```bash
+    make tunnel
+    ```
+
+1. Deploy your cluster
+    <!-- TODO (bv) need to create the dev-vnet for the cluster to be created in.  go run ./hack/cluster create expects this -->
+
+    ```bash
+    CLUSTER=mycluster go run ./hack/cluster create
```

Comment:
@jim-minter  @mjudeikis  do we want to update `RESOURCEGROUP` here, or is it fine if we create the OpenShift cluster in the rg that all the RP resources are in?

Reply:
You mean V4-stuff resource group? Or which one? I think it should be separate.